app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    new_one = command.split()
+    if len(new_one) != 3 and new_one[0] != "mv":
+        raise ValueError("Invalid command format")
+
+    move, first, second = new_one
```

Comment:
better to name vars understandable
```suggestion
    move, source, destination = new_one
```